Dockerfile

```diff
@@ -5,16 +5,18 @@ ENV LANG C.UTF-8
 WORKDIR /srv
 
 # System dependencies
-RUN apt-get update && apt-get install --yes python3-pip net-tools
-
-# Set git commit ID
-ARG COMMIT_ID
-RUN test -n "${COMMIT_ID}"
-RUN echo "${COMMIT_ID}" > version-info.txt
+# Node ppa provided by: https://github.com/nodesource/distributions
+RUN apt-get update && \
+    apt-get install curl --yes && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
+    apt-get update && \
+    apt-get install --yes build-essential python3-dev python3-pip git nodejs
 
 # Import code, install code dependencies
 ADD . .
 RUN pip3 install -r requirements.txt
+RUN npm install --global yarn
+RUN yarn build-prod
```

Comment:
Any building should ideally happen outside of the image itself, because the production image should only contain what it needs to run the application in production. In this case, I don't think we need `node` or `yarn` to actually run the app (as we usually don't with our sites).

So normally, we will expect the person building the image to run e.g. `./run build` first, and the we use an `ADD` instruction inside the image to copy in the built assets.

Could that also work here?

Reply:
Yep, I agree, updated.